src/runtime/pkg/katautils/config.go

```diff
@@ -1009,11 +1009,12 @@ func newClhHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		return vc.HypervisorConfig{}, err
 	}
 
-	if sharedFS != config.VirtioFS && sharedFS != config.VirtioFSNydus {
-		return vc.HypervisorConfig{}, errors.New("clh only support virtio-fs or virtio-fs-nydus")
+	if sharedFS != config.VirtioFS && sharedFS != config.VirtioFSNydus && sharedFS != config.VirtioFSNone {
```

Comment:
This is a bit awkward, in that we also are doing just about this same check up above in the map of supported shared-filesystem options in sharedFS.  I wonder if there's a way we can dedup/clean this up, so we don't have to make changes in multiple spots again next time.

Reply:
Ah, sorry - I see that's a 'general' check, whereas this is cloud hypervisor specific check.  meh, its a bit awkward/broken abstraction that we are checking clh specific bits in a generic katautils pkg, but I guess that isn't necessarily your fault.